DQM/src/DQM/SampleValidation.cxx

```diff
@@ -89,7 +87,7 @@ namespace dqm {
   }
 
   int SampleValidation::pdgid_label(const int pdgid) {
-    int label = 0;
+    int label = 18;
```

Comment:
Cool! Just add a comment that this is a catch-all/overflow bin for code readability